gnocchi/tests/indexer/sqlalchemy/test_migrations.py

```diff
@@ -67,6 +68,9 @@ def setUp(self):
 
     def _drop_database(self):
         try:
+            # We need to close any other connection before calling the database drop method.
+            self.index.get_engine().dispose()
```

Comment:
How is this related to the docs change?

Reply:
No this is not. However, it is needed to fix the CI.

Reply:
I added the fix in another PR then: https://github.com/gnocchixyz/gnocchi/pull/1485